Formula/mypy.rb

```diff
@@ -15,7 +15,6 @@ class Mypy < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "929f59cbdf068995a3caa9b52d241c76abd2bcf030ca3556da42ba65ca6dae20"
   end
 
-  depends_on "python-typing-extensions"
```

Comment:
Reverting #111646 since it causes a cyclic dependency

Reply:
Thanks!